server/validation/place.js

```diff
@@ -20,6 +22,8 @@ export default {
   },
   updatePlace: {
     body: {
+      accessibilityInfo: Joi.string().min(10).required(),
+      capacity: Joi.string().min(1).required(),
```

Comment:
`min(1)` should be already covered by `required()` as an empty string is considered an invalid request.

Reply:
https://github.com/adzialocha/hoffnung3000/pull/94/commits/5dd7a30f35553add1432d56c69618f2fd5617e4d